### PR TITLE
boac-5166 - Flight deck - application profile complete, remove alert logs export, service alert partially complete

### PR DESCRIPTION
**Pull request by mohamalnadi**
**Jiras:** 

- https://jira-secure.berkeley.edu/browse/BOAC-5160
- **sub task:** https://jira-secure.berkeley.edu/browse/BOAC-5163
- **sub task:** https://jira-secure.berkeley.edu/browse/BOAC-5165
- **sub task:** https://jira-secure.berkeley.edu/browse/BOAC-5166

### CODE REVIEW

File: src/components/admin/EditServiceAnnouncement.vue [hunk 1]
```diff
@@ -27,39 +27,42 @@
...
-          <b-btn
+          <v-btn
             id="button-update-service-announcement"
             :disabled="text === originalText"
             variant="primary"
             class="btn-primary-color-override mt-2"
             @click="updateText"
           >
-            <span v-if="isSaving"><font-awesome icon="spinner" spin /> Update...</span>
+            <!-- <span v-if="isSaving"><font-awesome icon="spinner" spin /> Update...</span> -->
```
Comment (pauline2k) [inline src/components/admin/EditServiceAnnouncement.vue:37]: @mohamalnadi Do we need another new-style spinner component to replace this commented-out line?

File: src/views/FlightDeck.vue [hunk 2]
```diff
@@ -22,11 +22,7 @@
       </div>
       <div class="mt-2 pt-5">
         <h2 id="manage-topics-header" class="page-section-header-sub">Manage Topics</h2>
-        <ManageTopics />
-      </div>
-      <div class="mt-2 pt-5">
-        <h2 class="page-section-header-sub">Alerts Log Export</h2>
-        <AlertsLogExport />
+        <!-- <ManageTopics /> -->
```
Comment (pauline2k) [inline src/views/FlightDeck.vue:25]: @mohamalnadi Commented-out ManageTopics here and below indicates work in progress?

**Comment by pauline2k** [comment]
@mohamalnadi Looks good to me! Just let me know if I'm correct in assuming the commented-out components are deliberately commented out to indicate work in progress.

**Comment by mohamalnadi** [comment]
@pauline2k - yes, everything that is commented out is because it is work in progress! thank you!

**Pull request merged**